internal/tiltfile/files.go

```diff
@@ -339,7 +339,22 @@ func (s *tiltfileState) helm(thread *starlark.Thread, fn *starlark.Builtin, args
 		return nil, fmt.Errorf("Argument 0 (path): %v", err)
 	}
 
-	yaml, err := s.execLocalCmdArgv("helm", "template", localPath.path)
+	templates, err := ioutil.ReadDir(filepath.Join(localPath.path, "templates"))
+	if err != nil {
+		return nil, fmt.Errorf("Error reading chart template directory at %s: %v", localPath.path, err)
+	}
+
+	cmd := []string{"helm", "template", localPath.path}
+	for _, t := range templates {
+		name := t.Name()
+		if name == "tests" || name == "NOTES.txt" || filepath.Ext(name) == ".tpl" {
```

Comment:
this seems kind of brittle. how confident are you that this is correct?

Reply:
poking at the code a bit, i think the actual rule is "ignore NOTES.txt and files that start with underscore"

https://github.com/helm/helm/blob/master/cmd/helm/template.go#L229

i wonder if we should also file a bug report upstream that `helm template` should skip tests by default

Reply:
Great find! I need to get better at just looking at what open source code does in cases like this.